voting/src/general_meeting.rs

```diff
@@ -186,6 +186,29 @@ impl TxOwner for Context {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum Error {
+    InvalidMeeting,
+}
+
+#[service]
+pub trait GeneralMeetingManager: Service {
+    fn get_meeting(&mut self, meeting_id: &GeneralMeetingId) -> Result<GeneralMeeting, Error>;
+}
+
+impl GeneralMeetingManager for Context {
+    fn get_meeting(&mut self, meeting_id: &GeneralMeetingId) -> Result<GeneralMeeting, Error> {
+        if !self.storage().has(meeting_id.as_ref()) {
+            return Err(Error::InvalidMeeting)
+        }
+        let meeting: GeneralMeeting = {
+            let bytes = self.storage().get(meeting_id.as_ref()).ok_or_else(|| Error::InvalidMeeting)?;
+            serde_cbor::from_slice(&bytes).expect("General meeting is serialized by this code")
+        };
```

Comment:
Since you checked the existence of the meeting using the `storage().has`, you don't need to check the error once more.
How about this:
```rust
             let bytes = self.storage().get(meeting_id.as_ref()).expect("We checked the existence in the above if statement");

```